<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="AddNewDealButton">
          <property name="text">
           <string>Добавить новую сделку</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="EditDealsButton">
          <property name="text">
           <string>Редактировать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showChart">
          <property name="text">
           <string>Отобразить график</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteDeal">
          <property name="text">
           <string>Удалить сделку</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updatePrice">
          <property name="text">
           <string>Обновить цены</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Прибыль/убыток </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="allPnL"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="assetsChart"/>
    <addaction name="AboutProgram"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="assetsChart"/>
   <addaction name="AboutProgram"/>
   <addaction name="separator"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="assetsChart">
   <property name="text">
    <string>Диаграмма активов</string>
   </property>
  </action>
  <action name="AboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
